releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -4,6 +4,99 @@
 # Microsoft.FeatureManagement.AspNetCore
 [Source code ][source_code_web] | [Package (NuGet)][package_web] | [Samples][samples_web] | [Product documentation][docs]
 
+## 4.0.0-preview - January 4, 2024
+
+### Variants
+
+Variants provide the ability to surface different variations of a feature, allowing different segments of an audience to get different variations of a feature, all within a single feature flag. Variants also contain a value that can either reference another ConfigurationSection or be declared inline on the flag.
```

Comment:
Repition of the phrase "variations of a feature" breaks the flow of the sentence for me.

Reply:
I took a look at the dynamic features opening section, it looks like it could have been used here without much modification. It also mentions one of the key points that a variant is not limited to boolean.

Reply:
Done. I adjusted it a little extra around the "Previously, this library only worked with feature flags." to try to separate "Feature Flags" from the idea of "flagging" or "boolean flags", since this library still "only works with Feature Flags", the proper-noun version.

Let me know if the adjusted version makes sense to you.